tests/SprykerEcoTest/Zed/UnzerApi/Business/UnzerApiFacade/PerformAuthorizeApiCallTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\UnzerApi\Business\UnzerApiFacade;
+
+use Generated\Shared\Transfer\UnzerApiErrorResponseTransfer;
+use SprykerEcoTest\Zed\UnzerApi\Business\UnzerApiFacadeBaseTest;
+
+/**
+ * @group Functional
+ * @group SprykerEco
+ * @group Zed
+ * @group UnzerApi
+ * @group Business
+ */
+class PerformAuthorizeApiCallTest extends UnzerApiFacadeBaseTest
+{
+    /**
+     * @var string
+     */
+    protected const FIXTURE_FILE_NAME = 'authorizeResponseBody.json';
+
+    /**
+     * @return void
+     */
+    public function testReturnsSuccessfulResponse(): void
+    {
+        // Arrange
+        $unzerApiRequestTransfer = $this->tester->createUnzerApiRequestTransfer();
+
+        // Act
+        $unzerApiResponseTransfer = $this->facade->performAuthorizeApiCall($unzerApiRequestTransfer);
+        $unzerApiAuthorizeResponseTransfer = $unzerApiResponseTransfer->getAuthorizeResponseOrFail();
+
+        // Assert
+        $this->assertTrue($unzerApiResponseTransfer->getIsSuccessful());
+        $this->assertNotEmpty($unzerApiAuthorizeResponseTransfer->getId());
+    }
+
+    /**
+     * @return void
+     */
+    public function testReturnsErrorResponse(): void
+    {
+        // Arrange
+        $unzerApiRequestTransfer = $this->tester->createUnzerApiRequestTransfer();
+        $this->returnSuccessResponse = false;
+
+        // Act
+        $unzerApiResponseTransfer = $this->facade->performAuthorizeApiCall($unzerApiRequestTransfer);
+
+        // Assert
+        $this->assertFalse($unzerApiResponseTransfer->getIsSuccessful());
+        $this->assertInstanceOf(UnzerApiErrorResponseTransfer::class, $unzerApiResponseTransfer->getErrorResponse());
```

Comment:
assertNotNull could be used instead
this property cannot be an instance of another class
or probably even more prices assert could be used

same for cases below